internal/mbsc/mbsc.go

```diff
@@ -85,6 +86,15 @@ func (m *mbsc) SetImageStatus(mbscObj *kmmv1beta1.ModuleBuildSignConfig, image s
 	mbscObj.Status.Images = append(mbscObj.Status.Images, imageState)
 }
 
+func (m *mbsc) GetImageStatus(mbscObj *kmmv1beta1.ModuleBuildSignConfig, image string, action kmmv1beta1.BuildOrSignAction) kmmv1beta1.BuildOrSignStatus {
+	for _, imageState := range mbscObj.Status.Images {
+		if imageState.Image == image && imageState.Action == action {
```

Comment:
Why do we need to pass the action to get the status?

Reply:
to make sure that we get the correct one. MIC is sending request for image and action, if we don't pass action, we might get the status for the previous action

Reply:
the status is updated atomically by MBSC controller, but it is updated once the MBSC controller has the result. MIC should never update the status of MBSC object.